agent/consul/internal_endpoint.go

```diff
@@ -43,7 +43,14 @@ func (m *Internal) NodeInfo(args *structs.NodeSpecificRequest,
 				return err
 			}
 
-			reply.Index, reply.Dump = index, dump
+			// old UIs may not handle .Dump so we added all peered nodes in .ImportedDump
```

Comment:
We can continue to assign all results to `reply.Dump` since an old UI would not populate `args.PeerName`